docs/status.md

```diff
@@ -139,7 +139,7 @@ one of the following tracking labels.
 | torch_index_select       | no            | revisit      | no             | no              | no          |
 | trace                    | no            | revisit      | no             | yes             | no          |
 | transpose                | yes           | yes          | yes            | yes             | yes         |
-| triangular_solve         | no            | yes*         | yes*           | no              | no          |
+| triangular_solve         | yes           | yes          | yes            | no              | no          |
```

Comment:
Let's open a ticket to drop `TRANSPOSE_INVALID` and change Verification to "revisit" until the ticket is fixed.

Reply:
Filed an issue here: #534